### DISCUSSION TITLE
Is "Zeit Now" now called "Vercel Now", or is it just called Vercel?

### DISCUSSION THREAD

**Discussion by simonw** [vercel/vercel#4129] [General] [answered]
Trying to figure out if my [datasette-publish-now](https://github.com/simonw/datasette-publish-now) plugin should actually be called `datasette-publish-vercel`,

**Comment by TooTallNate** [comment, marked as answer]
It's just "Vercel" 👍

**Reply by mcsdevv** [reply]
You can find more information about all the relevant changes [here](https://www.notion.so/Vercel-Launch-1cab2369ab1c4d8db1987b68708be683).
